nasa/openmct
# Update TelemetryTable config to allow disable multiselect

Related NIRVSS issue: https://babelfish.arc.nasa.gov/jira/browse/NIRVSSOMCT-178
Related NIRVSS PR: https://babelfish.arc.nasa.gov/bitbucket/projects/NIRVSSOMCT/repos/openmct_nirvss_client/pull-requests/44/overview

### Author Checklist
Check | Passed
-- | --
Changes address original issue? | NA 
Unit tests included and/or updated with changes? | NA 
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | Y

### Testing instructions
1. Create TelemetryTable
2. Add telemetry data point
3. Check all features such sorting, filter, play/pause, export, marking working as expected.
